LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/LoRaMessage/LoRaPayload.cs

```diff
@@ -192,5 +192,8 @@ public static uint InferUpper32BitsForClientFcnt(ushort payloadFcnt, uint fcnt)
             var fcntServerUpper = fcnt & MaskHigher16;
             return fcntServerUpper | payloadFcnt;
         }
+
+        public virtual bool RequiresConfirmation()
```

Comment:
can we make this a property?

Reply:
done d6f480766b705fe7d57b9d11f4d53d013d5b4ae9